pokemongo_bot/cell_workers/snipe_pokemon.py

```diff
@@ -0,0 +1,112 @@
+import json
+import os
+import re
+import sys
+import time
+
+from . import PokemonCatchWorker
+from base_task import BaseTask
+from pokemongo_bot import logger
+from utils import distance
+
+class SnipePokemon(BaseTask):
+
+    def initialize(self):
+        self.api = self.bot.api
+        self.pokemon_list = self.bot.pokemon_list
+        self.position = self.bot.position
+
+    def work(self):
+        if self.bot.tick_count is not 1:
+            return
+
+        while True:
```

Comment:
@TheSavior Or we could use a configuration file with list of snipe targets and process them one by one and remove the targets as they are processed. Would be usable even when bot is running.